ui/components/multichain/asset-picker-amount/asset-picker-modal/AssetList.tsx

```diff
@@ -116,19 +116,28 @@ export default function AssetList({
               <Box marginInlineStart={2}>
                 {token.type === AssetType.native ? (
                   <TokenListItem
-                    title={nativeCurrency}
+                    title={
+                      token.symbol === nativeCurrency
+                        ? nativeCurrency
+                        : token.symbol
+                    }
                     primary={
-                      primaryCurrencyProperties.value ??
-                      secondaryCurrencyProperties.value
+                      token.symbol === nativeCurrency
+                        ? primaryCurrencyProperties.value ??
+                          secondaryCurrencyProperties.value
+                        : undefined
                     }
                     tokenSymbol={
+                      // eslint-disable-next-line no-nested-ternary
                       useNativeCurrencyAsPrimaryCurrency
-                        ? primaryCurrency
+                        ? token.symbol === primaryCurrency
+                          ? primaryCurrency
+                          : token.symbol
                         : secondaryCurrency
```

Comment:
A non nested ternary might be easier to read here

Reply:
Or maybe

```
!useNativeCurrencyAsPrimaryCurrency ? secondaryCurrency
    : token.symbol === primaryCurrency ? primaryCurrency
    : token.symbol
```

because then all your returned values line up with the right hand side

Reply:
I like your suggestion but the auto line breaks makes it hard to read as well:
```
 useNativeCurrencyAsPrimaryCurrency === false
    ? secondaryCurrency
    : token.symbol === primaryCurrency
    ? primaryCurrency
    : token.symbol
```

I ended up pulling out the logic to fix the nested ternary lint error too. What do you think of the latest version (lines 80-86)?

Reply:
Works for me